### ISSUE TITLE
Doesn't seem to work for negative bar plots

### ISSUE THREAD

**Issue by moritz-gerster** [elide-b/starbars#8]
```Python
import starbars
import matplotlib.pyplot as plt

# Example data
categories = ['A', 'B', 'C']
values = [-10, -20, -15]
annotations = [('A', 'B', 0.01), ('B', 'C', 0.05)]

plt.bar(categories, values)

# Annotate significance
starbars.draw_annotation(annotations)

plt.show()
```

![image](https://github.com/user-attachments/assets/ce9fbdc0-baf8-4c1d-b341-c83537477eb4)

The second annotation is missing.

**Comment by elide-b** [comment]
Hey! Thank you for bringing this issue to me.
I have just fixed and published a new minor version (`1.3.1`) of starbars that should take this into consideration. 

Always here for further help :)

**Comment by moritz-gerster** [comment]
Great, thanks!
